src/App.tsx

```diff
@@ -1,14 +1,94 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { ChangeEvent, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
-import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+
+import { getTodos, getUser } from './api';
+import { TodoModal } from './components/TodoModal';
+import { User } from './types/User';
+
+enum StatusesSelect {
+  Active = 'active',
+  Completed = 'completed',
+}
 
 export const App: React.FC = () => {
+  const [initialTodos, setInitialTodos] = useState<Todo[]>([]);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [user, setUser] = useState<User | null>(null);
+  const [statusSelect, setStatusSelect] = useState('All');
```

Comment:
```suggestion
  const [statusSelect, setStatusSelect] = useState(StatusesSelect.All);
```
move this value to the enum